netty/src/main/java/io/grpc/netty/NettyServerStream.java

```diff
@@ -53,6 +58,13 @@
   private final NettyServerHandler handler;
   private final Http2Stream http2Stream;
   private final WriteQueue writeQueue;
+  private final Attributes attributes;
+
+  private static Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
+      Attributes.Key.named("remote-addr");
```

Comment:
Maybe `GrpcUtil` would be a good place for it? I would suggest refactoring it to accommodate new set of keys by putting existing `*_KEY` fields (of `Metadata.Key` type) into inner `GrpcUtil.Metadata` static class and moving `REMOTE_ADDR_KEY` and `SSL_SESSION_KEY` into `Grpc.StreamAttributes` container, accordingly.

@ejona86 @zhangkun83 what do you guys think?